lib/option_set_creator.rb

```diff
@@ -14,200 +14,306 @@ def create_all
-        es: I18n.t('database.option_sets.loan_type.community_solar', locale: 'es')
-      })
+
+    liquidity_loc = loan_type.options.find_or_create_by(value: 'liquidity_loc')
+    liquidity_loc.label_translations = {
+      en: I18n.t('database.option_sets.loan_type.liquidity_loc', locale: 'en'),
+      es: I18n.t('database.option_sets.loan_type.liquidity_loc', locale: 'es')
+    }
+    liquidity_loc.save
+
+    investment_loc = loan_type.options.find_or_create_by(value: 'investment_loc')
+    investment_loc.label_translations = {
+      en: I18n.t('database.option_sets.loan_type.investment_loc', locale: 'en'),
+      es: I18n.t('database.option_sets.loan_type.investment_loc', locale: 'es')
+    }
+    investment_loc.save
+
+    investment = loan_type.options.find_or_create_by(value: 'investment')
+    investment.label_translations = {
+      en: I18n.t('database.option_sets.loan_type.investment', locale: 'en'),
+      es: I18n.t('database.option_sets.loan_type.investment', locale: 'es')
+    }
+    investment.save
+
+    evolving = loan_type.options.find_or_create_by(value: 'evolving')
+    evolving.label_translations = {
+      en: I18n.t('database.option_sets.loan_type.evolving', locale: 'en'),
+      es: I18n.t('database.option_sets.loan_type.evolving', locale: 'es')
+    }
+    evolving.save
+
+    single_liquidity_loc = loan_type.options.find_or_create_by(value: 'single_liquidity_loc')
+    single_liquidity_loc.label_translations = {
+      en: I18n.t('database.option_sets.loan_type.single_liquidity_loc', locale: 'en'),
+      es: I18n.t('database.option_sets.loan_type.single_liquidity_loc', locale: 'es')
+    }
+    single_liquidity_loc.save
+
+    wc_investment = loan_type.options.find_or_create_by(value: 'wc_investment')
+    wc_investment.label_translations = {
+      en: I18n.t('database.option_sets.loan_type.wc_investment', locale: 'en'),
+      es: I18n.t('database.option_sets.loan_type.wc_investment', locale: 'es')
+    }
+    wc_investment.save
+
+    sa_investment = loan_type.options.find_or_create_by(value: 'sa_investment')
+    sa_investment.label_translations = {
+      en: I18n.t('database.option_sets.loan_type.sa_investment', locale: 'en'),
+      es: I18n.t('database.option_sets.loan_type.sa_investment', locale: 'es')
+    }
+    sa_investment.save
+
+    community_solar = loan_type.options.find_or_create_by(value: 'community_solar')
+    community_solar.label_translations = {
+      en: I18n.t('database.option_sets.loan_type.community_solar', locale: 'en'),
+      es: I18n.t('database.option_sets.loan_type.community_solar', locale: 'es')
+    }
+    community_solar.save
   end
 
   def create_public_level
     public_level = OptionSet.find_or_create_by(division: Division.root, model_type: Loan.name,
       model_attribute: 'public_level')
-    public_level.options.destroy_all
-    public_level.options.create(value: 'public', label_translations: {
+
+    public = public_level.options.find_or_create_by(value: 'public')
```

Comment:
I think `public` is a reserved word, if this works I wouldn't guarantee it would continue to in the future. I'd feel better about naming this variable `public_option` or something.